toffy/tiling_utils_test.py

```diff
@@ -469,16 +471,16 @@ def test_generate_tiled_region_fov_list_base(moly_path, moly_roi_setting,
 
         # define the center points sorted
         actual_center_points_sorted = [
-            (x, y) for x in np.arange(0, 10, 5) for y in list(reversed(np.arange(85, 105, 5)))
+            (x, y) for y in list(reversed(np.arange(95, 105, 5))) for x in np.arange(0, 20, 5)
```

Comment:
It seems like there's been a couple PRs where this has gotten changed. What's going on here?

Reply:
This change ensures that the center points we're testing against are sorted in MIBI coordinate order (specifically on the y-axis, which is decreasing from top to bottom).